src/test/resources/log4j2.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration status="INFO">
+    <Appenders>
+        <Console name="ConsoleAppender" target="SYSTEM_OUT">
+            <PatternLayout pattern="%d{HH:mm:ss.SSS} [%t] %-5level %logger{36} - %msg%n"/>
```

Comment:
The `%logger{36}` thing logs the fully-quallified name of the logger which is usually excessive. `%c{1}` would be a better fit for the tests.

`%logger{36}`:
```
08:56:34.965 [main] TRACE com.github.sdonkov.PrefixTree - Tree doesn't contain this word - aa
```

 `%c{1}`:
```
08:59:12.020 [main] TRACE PrefixTree - Tree doesn't contain this word - aa
```

See [here](https://logging.apache.org/log4j/2.x/manual/layouts.html#Patterns) for explanation of `%c{1}`.